hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/dog/PhysicalActivityTransformations.scala

```diff
@@ -0,0 +1,274 @@
+        if (hasHotWeather) rawRecord.getOptionalBoolean("pa_h_concrete") else None,
+      paHotWeatherOutdoorWood =
+        if (hasHotWeather) rawRecord.getOptionalBoolean("pa_h_wood") else None,
+      paHotWeatherOutdoorOtherHardSurface =
+        if (hasHotWeather) rawRecord.getOptionalBoolean("pa_h_other_hard") else None,
+      paHotWeatherOutdoorGrassOrDirt =
+        if (hasHotWeather) rawRecord.getOptionalBoolean("pa_h_grass_dirt") else None,
+      paHotWeatherOutdoorGravel =
+        if (hasHotWeather) rawRecord.getOptionalBoolean("pa_h_gravel") else None,
+      paHotWeatherOutdoorSand =
+        if (hasHotWeather) rawRecord.getOptionalBoolean("pa_h_sand") else None,
+      paHotWeatherOutdoorAstroturf =
+        if (hasHotWeather) rawRecord.getOptionalBoolean("pa_h_astro") else None,
+      paHotWeatherOutdoorOtherSurface = hotWeatherOtherSurface,
+      paHotWeatherOutdoorOtherSurfaceDescription =
+        if (hotWeatherOtherSurface.getOrElse(false)) rawRecord.getOptional("pa_h_other")
+        else None,
+      paHotWeatherSunExposureLevel = rawRecord.getOptionalNumber("pa_h_sun"),
+      // cold weather
+      paColdWeatherMonthsPerYear = coldMonths,
+      paColdWeatherDailyHoursOutside = coldDaily,
+      paColdWeatherOutdoorConcrete =
+        if (hasColdWeather) rawRecord.getOptionalBoolean("pa_c_concrete") else None,
+      paColdWeatherOutdoorWood =
+        if (hasColdWeather) rawRecord.getOptionalBoolean("pa_c_wood") else None,
+      paColdWeatherOutdoorOtherHardSurface =
+        if (hasColdWeather) rawRecord.getOptionalBoolean("pa_c_other_hard") else None,
+      paColdWeatherOutdoorGrassOrDirt =
+        if (hasColdWeather) rawRecord.getOptionalBoolean("pa_c_grass_dirt") else None,
+      paColdWeatherOutdoorGravel =
+        if (hasColdWeather) rawRecord.getOptionalBoolean("pa_c_gravel") else None,
+      paColdWeatherOutdoorSand =
+        if (hasColdWeather) rawRecord.getOptionalBoolean("pa_c_sand") else None,
+      paColdWeatherOutdoorAstroturf =
+        if (hasColdWeather) rawRecord.getOptionalBoolean("pa_c_astro") else None,
+      paColdWeatherOutdoorOtherSurface = coldWeatherOtherSurface,
+      paColdWeatherOutdoorOtherSurfaceDescription =
+        if (coldWeatherOtherSurface.getOrElse(false)) rawRecord.getOptional("pa_c_other")
+        else None,
+      paColdWeatherSunExposureLevel = rawRecord.getOptionalNumber("pa_c_sun")
+    )
+  }
+
+  /**
+    * Calculate the percentage of time a dog walks at a given pace.
+    * @param selectedPaceTypes The set of all paces a dog walks at.
+    * @param paceType The pace for which we want to calculate a percent value
+    * @param pacePercent The proportion of time a dog walks at the given pace (only defined when the pace was one of multiple values selected).
+    * @return A decimal value representing the percentage of time a dog walks at the given pace.
+    */
+  def transformPace(
+    selectedPaceTypes: Option[Array[String]],
```

Comment:
No need for this to be an `Option` based on the body. It can just be an `Array`, and the caller has to deal with it.